### ISSUE TITLE
Academic citation

### ISSUE THREAD

**Issue by emmanuelsearch** [RIOT-OS/riot-os.org#18]
It is not crystal clear what paper to cite, should someone cite RIOT in an academic context.
How about adding in the landing page's FAQ a question like:" I want to cite RIOT, which reference should I use?"
and indicate in the answer the IEEE IoT Journal ref from 2018?

Emmanuel Baccelli, Cenk Gündogan, Oliver Hahm, Peter Kietzmann, Martine Lenders, Hauke Petersen, Kaspar Schleiser, Thomas C. Schmidt, Matthias Wählisch, RIOT: An Open Source Operating System for Low-End Embedded Devices in the IoT, IEEE Internet of Things Journal, Vol. 5, No. 6, pp. 4428-4440, December 2018.

**Comment by chrysn** [comment]
In the R language ecosystem, there is this nice feature that you can `citation(x)` almost anything, and it works like a help function:

```
> library('ggplot2')
> citation('ggplot2')

To cite ggplot2 in publications, please use:

  H. Wickham. ggplot2: Elegant Graphics for Data Analysis.
  Springer-Verlag New York, 2016.

A BibTeX entry for LaTeX users is

  @Book{,
    author = {Hadley Wickham},
    title = {ggplot2: Elegant Graphics for Data Analysis},
    publisher = {Springer-Verlag New York},
    year = {2016},
    isbn = {978-3-319-24277-4},
    url = {https://ggplot2.tidyverse.org},
  }

> 
```

If we go a similar route and provide a `make citation` command or something to that effect, not only is that easy to update, it could also populate the website (which already takes input from the build system) and could also include citations pulled in via the `pkg` system.

**Comment by tcschmidt** [comment]
This starting point leads in three possible directions, I suppose:

**1.) Fundamental RIOT publications:** These are (i) the 2013 Infocom poster and (ii) the 2018 IEEE IoT Journal article. These two are static, there is no momentum of adding further.    

**2.) Ongoing scientific work of the RIOT core groups:** One might expect that RIOT-centric core research is interesting, but in fact publications don't go well together. While there is a lot of decent work, some publications are just too far below the bar. 

**3.) Collecting all RIOT-related publications:** An approach earlier taken by other groups to collect and list all papers that base on their tools or systems repeatedly turned out infeasible. There are just too many of them and one would need a person to actively chase ...

In summary, I don't see a burning use case of dynamically handling publication references ...
